src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        int randomNumber = random.nextInt(100);
+        String randomColor = ColorSupplier.getRandomColor();
+
+        Ball ball = new Ball();
+        ball.color = randomColor;
+        ball.number = randomNumber;
```

Comment:
Directly accessing the fields of the 'Ball' instance is not a good practice. The fields should be private and accessed via getters and setters to encapsulate the properties of the 'Ball' class.